pytorch_blade/src/ltc/disc_compiler/replay.cpp

```diff
@@ -145,15 +150,54 @@ void DumpProgramAndData(
       stack.emplace_back(ts_data->data());
     }
   }
+  DumpIValues(stack, path);
+}
+
+torch::jit::Module ConvertGraphToModule(
+    const std::shared_ptr<torch::jit::Graph>& graph) {
+  torch::jit::Module module("__torch__.PlaceholderModule");
+  module.register_attribute("training", torch::jit::BoolType::get(), true);
+  FusionOutputs(graph);
+  for (auto& value : graph->inputs()) {
+    value->setDebugName("arg_" + value->debugName());
+  }
+  torch::blade::create_method_from_graph(module, "forward", graph);
+  return module;
+}
+
+void DumpIValue(const at::IValue& ivalue, const std::string& fname) {
+  auto chars = torch::jit::pickle_save(ivalue);
```

Comment:
nit: maybe use [`pickle()` with `writer`](https://github.com/pytorch/pytorch/blob/7cd2fa1d388bf240cd33ff933dc120e74ebc2eb3/torch/csrc/jit/serialization/pickle.h#L19
) to save iavlues directly into file? In case of large model.

Reply:
Some error while using `pickle with writer` and `torch::jit::unpickle` APIs, I will do more debug and update in the next PR.